### PR TITLE
186 bug fix chart error calling tofixed on undefined

### LINKED ISSUE #186: [BUG] Fix chart error calling `toFixed` on `undefined`
https://predict2.bwh.harvard.edu/dpdash/charts/631e9b4f86f752922901598b

### PR DESCRIPTION
**Pull request by Elkrival**
There is an error when toFixed is called on an undefined value.

### CODE REVIEW

**Review by mikestone14 — COMMENTED** [review]
were you able to determine what it was trying to find that was undefined?

**Review by WMahoney09 — APPROVED** [review]
Looks good 👍 
Minor: it might be worth making this a helper, thinking of the [rule of threes](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))
```js
export const formatAsPercentage = (value = 0) => `${value.toFixed(0)}%`
```

**Review by mikestone14 — COMMENTED** [review]
I took the graph from the error on the staging site and used it locally and the chart rendered fine... I'm not sure exactly how this error is coming up on staging.

**Comment by Elkrival** [comment]
@mikestone14 That happened to me too, I copied the object from the console and ran it locally. I suspect that it may be a missing value or calculation, maybe from a new a site, because even the graph table worked. Before debugging this was merged https://github.com/AMP-SCZ/dpdash/pull/178

File: views/components/Graphs/BarGraph.jsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const BarGraph = ({ graph }) => {
           formatter={(value, name, props) => {
             const { payload } = props
 
-            return `${payload.counts[name]} (${value.toFixed(0)}%)`
+            return `${payload.counts[name]} (${value?.toFixed(0) || 0}%)`
```
Comment (mikestone14) [inline views/components/Graphs/BarGraph.jsx]: I agree with Will that we should extract the `value?.toFixed(0) || 0` logic to a helper function.

Comment (Elkrival) [reply views/components/Graphs/BarGraph.jsx]: https://github.com/AMP-SCZ/dpdash/pull/187/commits/e579de9e94552417b0180ab7686f020c96a2462e

**Comment by Elkrival** [comment]
> Looks good 👍 Minor: it might be worth making this a helper, thinking of the [rule of threes](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))
> 
> ```js
> export const formatAsPercentage = (value = 0) => `${value.toFixed(0)}%`
> ```

Good point, thanks 👍

**Pull request merged**